devtools/gradle/gradle-application-plugin/pom.xml

```diff
@@ -37,6 +37,7 @@
             <artifactId>quarkus-project-core-extension-codestarts</artifactId>
             <type>pom</type>
             <scope>test</scope>
+            <version>${project.version}</version>
```

Comment:
Isn't better to keep this specific artifact in build-parent instead?

Reply:
We could. But then we could do the same for the rest of dependencies on POMs, e.g. https://github.com/quarkusio/quarkus/blob/main/integration-tests/amazon-lambda-http-resteasy-reactive/pom.xml#L40